app.js

```diff
@@ -1,57 +1,60 @@
 // Player Name
-var diceNum = 1;

```

Comment:
try to avoid using var and use the ES6's let/const, unless its for a purpose. :)

Reply:
I was confusing which one I need to use.
As "var" is old style but still we can use it. However, I learned the var during our class. So I just followed it.
"let" can update the values "const" can`t update. will not be changed.  Thank you for the advice. I feel comfortable using the let and Const in future work.